tools/benchmark/cmd/watch.go

```diff
@@ -194,7 +194,7 @@ func doWatch(stream v3.Watcher, requests <-chan string) {
 
 func recvWatchChan(wch v3.WatchChan) {
 	for range wch {
-		if atomic.LoadInt32(&nrRecvCompleted) == int32(eventsTotal) {
+		if atomic.LoadInt32(&nrRecvCompleted) >= int32(eventsTotal) {
```

Comment:
@heyitsanthony Can you explain more?

```
./benchmark --endpoints 127.0.0.1:2379 --clients 64 --conns 64 watch --total 50000
```

This is 

```
watchTotal = 50000         // Total number of watch requests
watchedKeyTotal = 10000    // Total number of keys to be watched
watchPutTotal = 10000      // Total number of put requests

watchTotal / watchedKeyTotal = 5 watchers on each key
eventsTotal = 5 * watchPutTotal = 50000 total events
```

And `len(watched) == watchedKeyTotal`, so there's no undercount in this case? I might be doing my math wrong.

But I am seeing it's overcounting more than 10 times like:

```
 762789 / 50000 Booooooooooooooooooooooooooooooooo! 1525.58% -10s
```

Thanks!